lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -109,8 +111,8 @@ class TwilioConnectionTransport extends StateMachine {
       _dominantSpeaker: {
         value: options.dominantSpeaker
       },
-      _adaptiveSimulcast: {
-        value: options.adaptiveSimulcast
+      _enableLiveTranscription: {
```

Comment:
Should we create a new property instead of replacing `adaptiveSimulcast`?

Reply:
I moved adaptiveSimulcast to the top so they are in alphabetical order.